src/main/java/org/matsim/run/LausitzScenario.java

```diff
@@ -138,8 +157,17 @@ protected Config prepareConfig(Config config) {
 			throw new RuntimeException(e);
 		}
 
-		// TODO: recreate counts format with car and trucks
 
+		if (emissions == EmissionAnalysisHandling.PERFORM_EMISSIONS_ANALYSIS) {
+//		set hbefa input files for emission analysis
+			EmissionsConfigGroup eConfig = ConfigUtils.addOrGetModule(config, EmissionsConfigGroup.class);
+			eConfig.setDetailedColdEmissionFactorsFile(HBEFA_FILE_COLD_DETAILED);
+			eConfig.setDetailedWarmEmissionFactorsFile(HBEFA_FILE_WARM_DETAILED);
+			eConfig.setAverageColdEmissionFactorsFile(HBEFA_FILE_COLD_AVERAGE);
+			eConfig.setAverageWarmEmissionFactorsFile(HBEFA_FILE_WARM_AVERAGE);
+			eConfig.setHbefaTableConsistencyCheckingLevel(EmissionsConfigGroup.HbefaTableConsistencyCheckingLevel.consistent);
+			eConfig.setDetailedVsAverageLookupBehavior(EmissionsConfigGroup.DetailedVsAverageLookupBehavior.tryDetailedThenTechnologyAverageThenAverageTable);
```

Comment:
If only running AVERAGE vehicle types,  the LookupBehaviour could be set IMO to directlyAverage (or whatever its name is). Together with just setting the average tables, it makes a bit more clear, that you run an average analysis (which is totally fine!)